src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1302 @@
+module SoilBiogeochemDecompCascadeMIMICSMod
```

Comment:
This entire file is new, so to see what I did this far to SoilBiogeochemDecompCascadeBGCMod.F90, you will need to look at the changes from my second commit.

Reply:
I have drawn attention to a few items with my posts here in the PR. I have also included several TODO comments directly in the code where I sometimes seek clarification for something.

@wwieder feel free to look over everything and/or it may be a good time to set up a meeting to go over progress this far and identify the next steps.

Reply:
This note for reference:
This entire file `SoilBiogeochemDecompCascadeMIMICSMod.F90` is new, so to see how it now differs from `SoilBiogeochemDecompCascadeBGCMod.F90`, compare my first commit in this PR to my current commit in this PR, e.g.:
`git diff 9985a65 446b587` and search for mimics